sonar-csscustomrules-plugin/src/main/java/org/sonar/css/MyLessCustomRulesDefinition.java

```diff
@@ -31,6 +31,7 @@ public String repositoryName() {
 
   @Override
   public String repositoryKey() {
+    System.out.print("aaa");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)